bucket/ditto.json

```
@@ -1,15 +1,15 @@
 {
     "version": "3.24.238.0",
     "description": "An enhanced clipboard manager",
-    "homepage": "https://ditto-cp.sourceforge.io/",
+    "homepage": "https://github.com/sabrogden/Ditto",
```

```suggestion
    "homepage": "https://ditto-cp.sourceforge.io",
```

It's homepage is still hosted on SF.